test/test_helper.rb

```diff
@@ -23,6 +23,17 @@
 require "sidekiq/testing"
 Sidekiq::Testing.inline!
 
+require "minitest/reporters"
+Minitest::Reporters.use! [
+  # By default we show which tests we're executing as they happen, in addition
+  # to showing a summary at the end. If you'd like to have more compact "dot" style
+  # output as tests are running you can comment out the SpecReporter and enable
+  # the DefaultReporter instead.
+  Minitest::Reporters::SpecReporter.new(print_failure_summary: true),
+  # Minitest::Reporters::DefaultReporter.new(color: true),
```

Comment:
Two questions:
1. Is there a reason the default reporter is commented out? Is it just to show what other reporters are available?
2. I think `color` is set [by default](https://github.com/minitest-reporters/minitest-reporters/blob/e534374caa46c601c5ff1c87921b52e7b98ce765/lib/minitest/reporters/progress_reporter.rb#L93C1-L95), but I could be reading the source code incorrectly.

Reply:
@gazayas

1. The main reason that it's there and commented out is to make it easy for people to go back to what they're used to easily if they want to.
2. When I was testing locally I didn't get color output unless I passed that option. (Could be something weird with my setup.)

Reply:
@jagthedrummer what do you think about having a `DOT=1` env variable to configure the output instead of having to open and edit `test_helper.rb` (and make sure not to commit the changes)?

Reply:
Ooh, that's an interesting thought. I had primarily been thinking that downstream app developers would make the change and commit it in their own repo. But it could be handy to be able to toggle it via ENV var.

Reply:
@kaspth I went with the ENV var idea but used a var called `BT_TEST_FORMAT` that can be set with `"dots"` to trigger the classic format.

With `BT_TEST_FORMAT=dots`:

```
$ BT_TEST_FORMAT=DOTS rails test test/models/user_test.rb
🌱 Generating global seeds.
🌱 Generating test environment seeds.
Emptying /Users/jgreen/projects/bullet-train-co/bullet_train/test/reports

# Running tests with run options --seed 23930:

..

Finished tests in 0.228345s, 8.7587 tests/s, 8.7587 assertions/s.


2 tests, 2 assertions, 0 failures, 0 errors, 0 skips
Writing XML reports to /Users/jgreen/projects/bullet-train-co/bullet_train/test/reports
Coverage report generated for Unit Tests to /Users/jgreen/projects/bullet-train-co/bullet_train/coverage. 19 / 231 LOC (8.23%) covered.
```


Without `BT_TEST_FORMAT=dots`:

```
$ rails test test/models/user_test.rb
🌱 Generating global seeds.
🌱 Generating test environment seeds.
Emptying /Users/jgreen/projects/bullet-train-co/bullet_train/test/reports
Started with run options --seed 59171

UserTest
  test_details_provided_should_be_true_when_details_are_provided  PASS (0.18s)
  test_details_provided_should_be_false_when_details_aren't_provided PASS (0.07s)

Finished in 0.25666s
2 tests, 2 assertions, 0 failures, 0 errors, 0 skips
Writing XML reports to /Users/jgreen/projects/bullet-train-co/bullet_train/test/reports
Coverage report generated for Unit Tests to /Users/jgreen/projects/bullet-train-co/bullet_train/coverage. 19 / 231 LOC (8.23%) covered.
```

Reply:
@jagthedrummer nice, sounds good to me with `BT_TEST_FORMAT` 🙌

Reply:
:+1: